CHANGES.md

```diff
@@ -4,8 +4,8 @@
 
 ### New features
 
-* base: Pmac module version update, add motorPIGCS2 IOC and
-  add scaler IOC, and image.
+* base: Pmac module version update, add motorPIGCS2 IOC,
+  scaler IOC and MCA IOC, and images.
   by @guirodrigueslima in https://github.com/cnpem/epics-in-docker/pull/64
```

Comment:
Maybe splitting this into multiple items makes it easier to read. What do you think?